code/modules/admin/topic.dm

```diff
@@ -388,6 +388,28 @@
 			SSticker.real_reboot_time = 0 // If they set this at round end, show the "Reboot was cancelled by an admin" message instantly
 		href_list["secretsadmin"] = "check_antagonist"
 
+	else if(href_list["change_vip_target"])
+		if(!check_rights(R_ADMIN))
+			return
+		var/list/possible_targets = list()
+		for(var/datum/mind/possible_target in SSticker.minds)
+			if((possible_target != src) && ishuman(possible_target.current))
```

Comment:
Probably only minds in `SSticker.minds`. (If there's not, then that's a bigger issue)
```suggestion
		for(var/mind in SSticker.minds)
			var/datum/mind/possible_target = mind
			if((possible_target != src) && ishuman(possible_target.current))
```